.github/issue_template.md

```diff
@@ -0,0 +1,18 @@
+# Disclaimer
+If your Mafia or TourGuide version is wildy out of date when posting this issue, it may already be resolved and your issue may be closed. Please keep your things up to date!
+
+# Basic Data
+
+* KolMafia version:
+* TourGuide commit (`ash git_info("tourguide")`):
+* Class:
+* Run type (Casual/Normal/Hardcore/Aftercore):
+* Challenge Path (Standard/Path Name/None):
+
+# Current Behavior
+
+What is the current behavior? Do you know what caused it? Can you link the offending section of code?
+
+# Expected/Desired Behavior
+
+Please describe the behavior you are expecting. If the bug or feature involves a particular challenge path, item, quest, etc, please state that.
```

Comment:
i'm not sure the best way to set this up but what if we also add a way for them to categorize what their issue is about? like, maybe add one more row to basic data that's like... "Type of Issue (Bug/Feature/Upkeep)"?

Reply:
Honestly we should not merge the issue template actually. Looking into it, GitHub has a newer better issue template structure it can use now for templates for different types of issues. You can check it out under settings>general>features.

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

This is what I'm talking about. This wasn't a thing the last time I looked into templates.